Wikipedia/Code/TalkPageTopicNewViewController.swift

```diff
@@ -279,7 +223,7 @@ private extension TalkPageTopicNewViewController {
         bodyContainerVerticalPaddingConstraints.forEach { contentFittingBodyContainerHeight += $0.constant  }
         
         var availableVerticalScreenSpace = talkPageScrollView.frame.height - bodyContainerOrigin.y
-        availableVerticalScreenSpace = availableVerticalScreenSpace - finePrintContainerView.frame.height - beKindContainerView.frame.height
+        availableVerticalScreenSpace = availableVerticalScreenSpace - finePrintContainerView.frame.height - beKindInputAccessoryView!.frame.height
```

Comment:
Might be helpful to make `beKindInputAccessoryView` a lazy, non optional `var` to avoid having to force unwrap here

Reply:
Saw it was changes welcome so I made this change, feel free to revert